maxi-blocks/maxi-blocks
# Issue 3295 - fix image size and crop options

# Description
In image-maxi imageData was no longer in props, as we got rid of HOC which did that. And in background image ImageCropControl was passed wrong mediaID. Also fixed ImageCropControl as it was acting weird.

Fixes #3295 and #3198 

# How to test?
Check image size is appearing in dimension tab. Check crop control loads in background control. And check if crop control works.


# Test checklist



**_Front/Back Testing_**

-   [x] Check image size is appearing in dimension tab
-   [x] Check crop control loads in background control
-   [x] Test crop control

**_Pre-Code Testing_**

-   [x] Check image size is appearing in dimension tab
-   [x] Check crop control loads in background control
-   [x] Test crop control

# Checklist
